src/app/pages/OpenWalletPage/Features/FromPrivateKey/index.tsx

```diff
@@ -59,14 +61,28 @@ export function FromPrivateKey(props: Props) {
           htmlFor="privateKey"
           error={privateKeyIsValid === false ? t('openWallet.privateKey.error', 'Invalid private key') : ''}
         >
-          <Box border={false}>
-            <TextArea
+          <Box border={false} direction="row" round="small">
+            <TextInput
```

Comment:
This removes newlines, and that breaks importing multiline private keys like
```
-----BEGIN ED25519 PRIVATE KEY-----
ZqtrV0QtEY/JemfTPbOl9hgk3UxHXfZO42G4sG+XKHThZTM+GvRiqsAgc7magKNN
4MEkyO0pi7lJeunILQKiZA==
-----END ED25519 PRIVATE KEY-----
```

I'd prefer TextArea but it doesn't have type=password. And Firefox doesn't have `-webkit-text-security`. So I guess parseKey can be fixed with:
```diff
      const keyWithoutEnvelope = key
-       .replace(/^-----.*$\n?/gm, '')
        .replace(/\n/gm, '')
        .replace(/ /g, '')
+       .replace(/^-----.*?-----/, '')
+       .replace(/-----.*?-----$/, '')
```